docs/source/en/autoclass_tutorial.md

```diff
@@ -141,3 +142,24 @@ Easily reuse the same checkpoint to load an architecture for a different task:
 Generally, we recommend using the `AutoTokenizer` class and the `TFAutoModelFor` class to load pretrained instances of models. This will ensure you load the correct architecture every time. In the next [tutorial](preprocessing), learn how to use your newly loaded tokenizer, image processor, feature extractor and processor to preprocess a dataset for fine-tuning.
 </tf>
 </frameworkcontent>
+
+## AutoBackbone
```

Comment:
This guide is aimed at beginners, so it would be great to let them know when (or why) to use AutoBackbone.